lib/sshkit.ex

```diff
@@ -90,13 +99,26 @@ defmodule SSHKit do
   hosts = ["10.0.0.1", "10.0.0.2"]
   context = SSHKit.context(hosts)
   ```
+
+  Any shared options can be specified in the second argument.
+  Here we add a user and port for all hosts.
+
+  ```
+  hosts = ["10.0.0.1", "10.0.0.2"]
+  options = [user: "admin", port: "2222"]
+  context = SSHKit.context(hosts, options)
+  ```
   """
-  def context(hosts) do
+
+  def context(hosts, options \\ []) do
```

Comment:
we should probably name the options param `shared_options` (as in `host`) since they are shared :)